deploy/examples/nfs.yaml

```diff
@@ -4,14 +4,24 @@ metadata:
   name: my-nfs
   namespace: rook-ceph # namespace:cluster
 spec:
+  # For Ceph v16, use the block here.
+  # rados settings aren't necessary in Ceph Versions equal to or greater than Pacific 16.2.7
   rados:
-    # The Ganesha pool spec. Must use replication.
     poolConfig:
```

Comment:
Should the `poolConfig` be under `rados`? If so, the comment above on line 8 is confusing about how the rados settings aren't necessary now.

Reply:
I suppose we do have time to change this before cutting the 1.8 release since it was added recently by Seb. IMO, I would put `poolConfig` on the top level, like you are suggesting. I think having a `rados` block seems a little too specific. Let's bring @leseb into that discussion.

Reply:
Oh right, we still have time before cutting 1.8. I guess this used to be valid while the RadosSpec made sense, but not anymore. +1 from moving to the top-level.

Reply:
I also wonder whether we should have this here or not. If there are multiple CephNFS specs, they may contend with each other to set different replication settings on the same `.nfs` pool. Maybe we should just remove this config and allow `.nfs` to be managed by @travisn 's suggestion here: https://github.com/rook/rook/issues/9209#issuecomment-974477475

Reply:
I'm not sure, but I agree the multiple CR is a concern. However, is this a valid use case? Should we support it? Especially if they all end up using the same pools.
I'm not really keen on leaving the pool management to the user, the nfs implementation is already complex enough (at least the migration).